homeassistant/components/tibber/strings.json

```diff
@@ -13,7 +13,7 @@
         "data": {
           "access_token": "[%key:common::config_flow::data::access_token%]"
         },
-        "description": "Enter your access token from https://developer.tibber.com/settings/accesstoken"
+        "description": "Enter your access token from https://developer.tibber.com/settings/access-token"
```

Comment:
The link should be moved out of the translations and be added as a placeholder (links shouldn't be translated).